$NetBSD$

On PaX mprotect systems, the test binary would need to be marked with
"paxctl +m". Disable test.

--- meson.build.orig	2020-03-28 05:38:50.000000000 +0000
+++ meson.build
@@ -232,13 +232,6 @@ int main(void) {
     args: debug_arg, dependencies: spidermonkey,
     name: 'SpiderMonkey sanity check')
 
-if not minimal_program.compiled() or minimal_program.returncode() != 0
-    error('''A minimal SpiderMonkey program
-could not be compiled, linked, or run. Most likely you should build it with a
-different configuration. Check the recommended configuration:
-https://github.com/spidermonkey-embedders/spidermonkey-embedding-examples/blob/esr60/docs/Building%20SpiderMonkey.md''')
-endif
-
 have_printf_alternative_int = cc.compiles('''
 #include <stdio.h>
 int main(void) {
